### PR TITLE
refactor: ordinal modal

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/4225987130).

Screen Shot 2023-02-20 at 1 13 55 PM

### CODE REVIEW

File: src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx [hunk 1]
```diff
@@ -32,27 +33,51 @@ export function ReceiveCollectibleOrdinal() {
           <Title fontSize="20px" lineHeight="28px" px="extra-loose">
             Deposit Ordinal inscription
           </Title>
-          <ErrorLabel>Important to keep in mind</ErrorLabel>
-          <Text color={color('text-title')} fontSize={1} px={['base', '40px']}>
-            This Taproot address is for Ordinals only. Do not use this address more than once for
-            depositing an inscription
+          <Text color={color('text-title')} fontSize={1} px={['50px', '70px']}>
+            Use your wallet to create inscriptions within minutes in
```
Comment (edu-stx) [inline src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: > Use your wallet to create insciptions [...] *in*?

Maybe

> Use your wallet to create insciptions [...] *with*?

**Review by edu-stx — APPROVED** [review]

File: src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx (hunk 1 above)
Comment (fbwoolf) [reply src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: @mica000 or @markmhx what do you think here? I also paused on this ...could use `here` too? I do think `in` seems off?

Comment (kyranjamie) [reply src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: Wouldn't it be `on gamma.io`?

File: src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx [hunk 2]
```diff
@@ -32,27 +33,51 @@ export function ReceiveCollectibleOrdinal() {
...
+              color={color('accent')}
+              fontSize={1}
+              isInline
+              onClick={() => openInNewTab('https://ordinalsbot.com/')}
+              spacing="extra-tight"
+              textDecoration="underline"
+              type="button"
+            >
+              <Text color="inherit">ordinalsbot.com</Text>
+              <Box as={FiArrowUpRight} />
```
Comment (kyranjamie) [inline src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: Isn't this the same as `<FiArrowUpRight />`

File: src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx [hunk 3]
```diff
@@ -32,27 +33,51 @@ export function ReceiveCollectibleOrdinal() {
...
-            type="button"
-          >
-            Go back
-          </Button>
-          <PrimaryButton flexGrow={1} onClick={copyToClipboard}>
-            Proceed to copy address
-          </PrimaryButton>
-        </Flex>
+        <PrimaryButton flexGrow={1} my="extra-loose" onClick={copyToClipboard} width="100%">
+          Proceed to copy address
```
Comment (kyranjamie) [inline src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: Imo this is too verbose, shouldn't it just be Copy address? @markmhx

Comment (markmhendrickson) [reply src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: yep shorter works for me

File: src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx (hunk 1 above)
Comment (markmhendrickson) [reply src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx]: ...with these apps:

**Pull request merged**